extensions/resteasy-classic/resteasy/deployment/src/main/java/io/quarkus/resteasy/deployment/ResteasyServletProcessor.java

```diff
@@ -53,9 +53,8 @@ public void jaxrsConfig(
             BuildProducer<io.quarkus.resteasy.server.common.spi.ResteasyJaxrsConfigBuildItem> resteasyJaxrsConfig,
             HttpRootPathBuildItem httpRootPathBuildItem) {
         if (resteasyServerConfig.isPresent()) {
-            String rp = resteasyServerConfig.get().getRootPath();
-            String rootPath = httpRootPathBuildItem.resolvePath(rp.startsWith("/") ? rp.substring(1) : rp);
-            String defaultPath = httpRootPathBuildItem.resolvePath(resteasyServerConfig.get().getPath());
```

Comment:
I think this was completely incorrect and only worked because `resteasyServerConfig.get().getPath()` was always starting with a '/' and thus resolving just returned it.

That being said, this is all very confusing and unclear and the very generic variable names don't help to understand what these things are supposed to be.